main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
```

Comment:
This is an "ansible" fix, not a "terraform" fix. Terraform will execute all local-exec provisioners almost immediately. This will tie up all the threads. We don't want this. We need to make terraform aware that it shouldn't run the local-exec provisioner until the remote-exec provisioner has executed.